exiftool.go

```diff
@@ -155,17 +187,34 @@ func (e *Exiftool) ExtractMetadata(files ...string) []FileMetadata {
 			continue
 		}
 
+		fms[i].Err = e.log("sending to exiftool STDIN:")
+		if fms[i].Err != nil {
+			continue
+		}
+
 		for _, curA := range extractArgs {
+			e.logf("\t%s\n", curA)
```

Comment:
The error that might be returned by `e.logf` is not stored in `fms[i].Err` (like any other logging call)

Reply:
Ah, I missed that one, thanks. I'll update it.

Reply:
done